src/views/Project/index.jsx

```diff
@@ -0,0 +1,247 @@
+            'description'
+          ).sort((a, b) => (a.lastupdate > b.lastupdate ? -1 : 1)),
+          error,
+          loading,
+        });
+      });
+  }
+
+  fetchBugs(variables) {
+    const { client } = this.props;
+
+    client
+      .query({
+        query: Bugs,
+        variables,
+        context: { link: 'bugzilla' },
+      })
+      .catch(
+        () =>
+          new Promise(resolve => {
+            resolve(false);
+          })
+      )
+      .then(({ data, loading, error }) => {
+        const bugsData = data.bug.edges.map(edge => edge.node).map(bug => ({
+          assignedto: bug.assignedTo.name || 'None',
+          project: bug.component,
+          id: bug.id,
+          tag: bug.tags || '',
+          description: `${bug.id} - ${bug.summary}`,
+          lastupdate: bug.lastChanged,
+        }));
+
+        this.setState({
+          data: _.uniqBy([...this.state.data, ...bugsData], 'description').sort(
+            (a, b) => (a.lastupdate > b.lastupdate ? -1 : 1)
+          ),
+          loading,
+          error,
+        });
+      });
+  }
+  fetchBugzillaDataNext() {
+    const { productList, productComponentList } = this.state;
+    const variables = {
+      searchProduct: {
+        products: productList,
+        tags: ['good-first-bug'],
+        statuses: ['NEW', 'UNCONFIRMED', 'ASSIGNED', 'REOPENED'],
+      },
+      paging: {
+        page: 0,
+        pageSize: 100,
+      },
+    };
+
+    // fetch bugs for products with no components
+    this.fetchBugs(variables, 'products');
+
+    // fetch bugs for each product that has components
+    productComponentList.forEach(item => {
+      variables.searchProduct.products = item.products;
+      variables.searchProduct.components = item.components;
+
+      this.fetchBugs(variables, item.products);
+    });
+  }
+
+  render() {
+    const { projectInfo, loading, error, data } = this.state;
+
+    return (
+      <div>
+        <header>
+          <Grid
+            container
+            spacing={16}
+            direction="column"
+            justify="center"
+            alignItems="center">
+            <Grid item xs={12}>
+              <div>
+                <div />
+                <div>
+                  <Typography variant="display4" align="center">
+                    {projectInfo.name}
+                  </Typography>
+                  {projectInfo.description && (
+                    <Typography align="center">
+                      <ReactMarkdown source={projectInfo.description} />
+                    </Typography>
+                  )}
+                </div>
+                <div />
+              </div>
+            </Grid>
+          </Grid>
+          <BugsTable
+            projectName={projectInfo.name}
+            error={error}
+            loading={loading}
+            data={data}
+          />
+        </header>
+      </div>
+    );
+  }
+}
+
+const ProjectClient = props => (
+  <ApolloConsumer>
```

Comment:
Wrap the entire app in the root component in an ApolloProvider, then you can use the graphql decorator.